pom.xml

```diff
@@ -385,6 +390,11 @@
                 <version>${project.version}</version>
                 <type>zip</type>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.inbound.auth.oauth2</groupId>
```

Comment:
This should not be possible as this could lead to a cyclic dependencies issue. Shall we try to remove this?

Reply:
Resolved